src/System.Security.Cryptography.Hashing.Algorithms/tests/MD5Tests.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+
+namespace System.Security.Cryptography.Hashing.Algorithms.Tests
+{
+    public class MD5Tests : HashAlgorithmTest
+    {
+        protected override HashAlgorithm Create()
+        {
+            return MD5.Create();
+        }
+
+        // Test cases are defined in RFC 1321, section A.5
+
+        [Fact]
+        public void MD5_Rfc1321_1()
+        {
+            Verify("", "d41d8cd98f00b204e9800998ecf8427e");
```

Comment:
As I mentioned offline, we could consider using [Theory]/[InlineData] for these, but I understand we're currently limited with the internal test harness.